microservice/src/main/java/nearsoft/skytouch/microservice/repository/ProductRepository.java

```diff
@@ -0,0 +1,10 @@
+package nearsoft.skytouch.microservice.repository;
+
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ProductRepository {
```

Comment:
Do you need to annotate both the interface and the implementation as @Repository ?